heron_base/launch/base.launch

```diff
@@ -32,23 +32,25 @@
     <group unless="$(optenv HERON_IMU_UM6 0)">
       <!-- newer herons use the microstrain CM5-25 IMU -->
       <group if="$(eval arg('namespace') == '')">
-        <include filename="$(find ros_mscl)/launch/microstrain.launch">
+        <include file="$(find ros_mscl)/launch/microstrain.launch">
           <arg name="name"          value="cv5" />
```

Comment:
Could name just be changed to `imu` to allow for any microstrain to be used?

Reply:
For practical purposes the name is purely cosmetic, so yeah. Changing it is no problem.  For the heron I believe they always use the CV5 as it's the easiest to waterproof.